### ISSUE TITLE
Not seeing any sideline information

### ISSUE THREAD

**Issue by chookity-pokk** [emacs-sideline/sideline#16]
I essentilly just copy/pasted the config info from the readme and installed sideline as well as sideline-lsp and sideline-flyckeck but when I use it in a buffer with lsp-mode enabled I don't get any information. When enabling lsp-ui-sideline though I can see the information. I've also tried enabling/disabling `sideline-mode` manually and not seen anything in the sideline. I am using emacs 29.1 and using Python's ruff lsp.

Here is my config

```
(setq lsp-ui-sideline-enable nil)  ; disable lsp-ui-sideline

(use-package sideline
  :init
  (setq sideline-backends-left-skip-current-line t   ; don't display on current line (left)
        sideline-backends-right-skip-current-line t  ; don't display on current line (right)
        sideline-order-left 'down                    ; or 'up
        sideline-order-right 'up                     ; or 'down
        sideline-format-left "%s   "                 ; format for left aligment
        sideline-format-right "   %s"                ; format for right aligment
        sideline-priority 100                        ; overlays' priority
        sideline-display-backend-name t              ; display backend name
	sideline-backend-left '(sideline-flycheck)   ; flyckec info on the bottom
	sideline-backend-right '(sideline-lsp)))     ; lsp info on the top
(add-hook 'lsp-mode 'sideline-mode)
```

I also checked the Messages buffer and don't see any errors there related to sideline or the lsp. I also uninstalled and re-installed all three packages and still get the same results.

**Comment by jcs090218** [comment]
Have you tried hovering errors and stuff? 🤔

**Comment by chookity-pokk** [comment]
I have, yes. I just tried again with a basic error and it unfortunately didn't show anything in the sideline.

**Comment by jcs090218** [comment]
Sorry, I mis-spelled the variable `sideline-backend-right`; it should be `sideline-backend[s]-right` with the `s`. So do the `sideline-backend-left`. 😓

**Comment by chookity-pokk** [comment]
Of course it was something super simple! Problem solved and thank you for the help!

**Comment by azimut** [comment]
Got a similar issue. I edebug it to what I think is the forced use of `sideline-truncate` which makes it return `nil` if unset and shortcircuits the `when-let*`.

https://github.com/emacs-sideline/sideline/blob/baa38cf8bae3d2eedc09df960fcb5049095e323b/sideline.el#L595

**Comment by jcs090218** [comment]
Fixed in b329d097a37585cd21c3b9a5862e0bb2a2f76442.
